safechef/safechef.jsonld

```diff
@@ -0,0 +1,23 @@
+  "certificateNo": "8460c404-c0a5-4653-aeae-44d03a1a7334",
+  "examDate": "2011-11-11",
+  "testCode": "df52b514-e13a-42b7-8030-3f928a27d167"
```

Comment:
We might want to look at Open Badges v3 and see how they would model this.

Reply:
K. Happy to take a look.

Reply:
/cc @kayaelle @dmitrizagidulin thoughts? Is there a "Workforce Skills" example in OBv3 that we could re-use for this example credential?